freqtrade/rpc/local_rest_server.py

```diff
@@ -0,0 +1,113 @@
+import threading
+import logging
+import json
+
+from flask import Flask, request
+from flask_restful import Resource, Api
+from json import dumps
+from freqtrade.rpc.rpc import RPC, RPCException
+
+
+logger = logging.getLogger(__name__)
+
+class LocalRestSuperWrap(RPC):
+    """
+    This class is for REST cmd line client
+    """
+    def __init__(self, freqtrade) -> None:
+        """
+        Init the LocalRestServer call, and init the super class RPC
+        :param freqtrade: Instance of a freqtrade bot
+        :return: None
+        """
+        super().__init__(freqtrade)
+        """ Constructor
+        :type interval: int
+        :param interval: Check interval, in seconds
+        """
+        self.interval = int(1)
```

Comment:
the int() cast is not necessary. 1 is already an integer:
```
>>> isinstance(1, int)
True
```

Reply:
done